dietpi/func/dietpi-set_hardware

```diff
@@ -2259,6 +2259,96 @@ _EOF_
 
 	}
 
+	#/////////////////////////////////////////////////////////////////////////////////////
+	# Dedicated GPU
+	# INPUT_DEVICE_VALUE
+	# - Uninstall
+	# - Intel
+	# - Nvidia
+	# - AMD
+	#/////////////////////////////////////////////////////////////////////////////////////
+	DedicatedGPU_Main(){
+	
+		#Current GPU
+		local gpu_current=$(sed -n '/^[[:blank:]]*CONFIG_GPU_DRIVER=/{s/^[^=]*=//p;q}' /boot/dietpi.txt)
+		gpu_current=${gpu_current,,}
```

Comment:
I first though about doing a fallback to custom/noop when calling this script, but actually its easier to do it here:
```suggestion
		[[ $gpu_current ]] && gpu_current=${gpu_current,,} || gpu_current='custom'
```